apps/site/assets/ts/stop/components/sidebar/Parking.tsx

```diff
@@ -44,6 +44,10 @@ const renderCapacity = (lot: ParkingLot): ReactElement<HTMLElement> => {
             <strong>Accessible spaces: </strong>
             {accessible}
           </li>
+          <li>
```

Comment:
Since overnight is optional and might not be in the data, we should show this HTML segment if overnight is available.
you can do something like
```{ overnight && <li><strong>Overnight Parking: ... </li> }```, where the false case {undefined} wouldn't render anything via jsx in React

It looks like the other lis in this file could use that treatment too since they are listed as optional, so you could fix those up too 🙏 .

Unless overnight data is always present, then we should change the interface to say `overnight` instead `overnight?`

Reply:
Sorry, I missed the part where we were adding a default value for no data.